components/menu/MenuItem.js

```diff
@@ -24,28 +23,30 @@ const factory = () => {
     };
 
     handleClick = event => {
-      if (this.props.onClick && !this.props.disabled) {
-        this.props.onClick(event, this);
+      const { disabled, onClick } = this.props;
+
+      if (onClick && !disabled) {
+        onClick(event, this);
       }
     };
 
     render() {
-      const { icon, caption, children, shortcut, selected, disabled, ...others } = this.props;
+      const { icon, caption, children, className, shortcut, selected, disabled, ...others } = this.props;
 
-      const className = cx(
+      const classNames = cx(
```

Comment:
The way classnames are being used is lacking consisteny and I'm crying inside.
Here it's `classNames`, while...

1. https://github.com/teamleadercrm/teamleader-ui/blob/master/components/island/Island.js#L34  (`classes`)
2. https://github.com/teamleadercrm/teamleader-ui/blob/master/components/popover/Popover.js#L123 (`customClassNames`)
3. https://github.com/teamleadercrm/teamleader-ui/blob/master/components/radio/RadioButton.js#L65 (`_className` (which is weird that nobody, including me, didn't notice this underscore syntax before))


  Please...

Reply:
I think we should just use `classNames` everywhere (that's also the reason that we import the `classNames` module as `cx` I guess).